### ISSUE TITLE
Not able to build with Unity 2018.4 

### ISSUE THREAD

**Issue by birdkingz** [playgameservices/play-games-plugin-for-unity#2619]
I'm using gradle to build my project on Unity 2018.4 and I keep getting this error
**transformDexArchiveWithExternalLibsDexMergerForRelease**
**android.support.v4.app.INotificationSideChannel**

After removing this plugin then I can build the project without any error.
Can anyone confirm whether the GooglePlayGamesPlugin-0.9.64 works on Unity 2018.4?

PS: I've removed most of the third party plugins which might cause a conflict yet I'm still facing this error while building.

**Comment by birdkingz** [comment]
I've been rolling back to old builds until 0.9.53 then I'm able to build without the above dex error.
Not sure what has been added in 0.9.6x to cause the error.

**Comment by handcircus** [comment]
I had this issue, in the end managed to resolve by installing google play resolver 1.2.122 https://github.com/googlesamples/unity-jar-resolver/releases/tag/v1.2.122 . 

Make sure you enable Jetify

Would be great if they bundled this version with the GPG plugin .

**Comment by Robbware** [comment]
@handcircus Did you simply force a resolve with that version of the google play resolver or did it work out of the box? Having issues with this.

**Comment by handcircus** [comment]
I think I spoke to soon 🤔 took a bit more messing around.
The problem seems to be “android-support-v4.jar". Anything that requires it
seems to cause issues. I nuked all the android plugins (jar/aar) we had and
started fresh. Latest FB seems to not require android-support-v4.jar, but
for another plugin I had to decompile and remap to an androidx version of
FileProvider. The shift to androidx seems to have caused a bunch of problems
